### ISSUE TITLE
Sweep past flight data into backbone.

### ISSUE THREAD

**Issue by bw4sz** [weecology/BOEM#5]
Its not clear yet if its best to retrain the entire dataset each time, or finetune on a per-flight dataset. Retraining takes longer and finetune allows the model to learn the label distribution of that current flight. Finetune may lead to catastrophic forgetting, and it risks overfitting. Its also awkward as we add in more labels.

**Comment by ethanwhite** [comment]
One possibility could be a bit of both where we using finetuning for short-term iteration but also do a full retrain periodically (once/week, once/month, every n new labels)

**Comment by bw4sz** [comment]
completed.
